impeller/display_list/dl_unittests.cc

```diff
@@ -447,6 +447,21 @@ TEST_P(DisplayListTest, IgnoreMaskFilterWhenSavingLayer) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(DisplayListTest, SaveLayerWithOpacityAndAdvancedBlendMode) {
```

Comment:
If you make this an Aiks unittest I believe we can get a golden test image out of it so it gets verified on CI.

Reply:
actually, that would only show an issue if it were an arm64 mac. Still, might be good to do.

Reply:
> If you make this an Aiks unittest I believe we can get a golden test image out of it so it gets verified on CI.

Sounds good, done.